webots_ros2_control/CMakeLists.txt

```diff
@@ -1,6 +1,17 @@
 cmake_minimum_required(VERSION 3.5)
 project(webots_ros2_control)
 
+# Check which ROS distribution is used, ros2control depends of that
```

Comment:
Would it make sense to add galactic?

Reply:
Yes we can, but as the code is written there is changes when we use `foxy` or `rolling` main release.

Thus in any other case the code will be as for `rolling` testing release (same as `Galactic`).